templates/layout.html.twig

```diff
@@ -24,9 +24,11 @@ Enjoy your theme!
 <div class="wrapper">
 
     {% if tabler_bundle.isLayoutHorizontal() %}
-        {% include '@Tabler/includes/layout_horizontal.html.twig' %}
+        {% use '@Tabler/includes/layout_horizontal.html.twig' %}
+        {% block layout_horizontal %}{{ parent() }}{% endblock %}
     {% elseif tabler_bundle.isLayoutVertical() %}
-        {% include '@Tabler/includes/layout_vertical.html.twig' %}
+        {% use '@Tabler/includes/layout_vertical.html.twig' %}
+        {% block layout_vertical %}{{ parent() }}{% endblock %}
```

Comment:
Changing `include` to `use`.

It allow devs to override blocks in `layout_horizontal` or `layout_vertical`.

#### Exemple
```twig
{% extends '@Tabler/layout.html.twig' %}

{% block navbar_search %}{# Disable search bar #}{% endblock %}
...
...
...
```

Reply:
Sure this works with the if?  Afaik the latest defined block wins and (from here I am unsure) the use should be evaluated during compile time, while the if is executed during runtime.

Reply:
I mean, I've tried both layout and it works with both of them.

The thing that I'm not sure that is working is the auto-completion/detection.

What IDE are you using ? Mine is PHPStorm and I don't know why but your Bundle with Tabler is not recognized by it.
AdminLTEBundle work like a charm, but not that one. 
How do you declare the `@Tabler` isn't it from the `xxxxBundle.php`

TablerBundle:
![image](https://user-images.githubusercontent.com/25293190/144870914-17b320d4-d8a0-4c40-8749-8ea8d4e54776.png)

AdminLTEBundle:
![image](https://user-images.githubusercontent.com/25293190/144871224-83cd6710-4853-4d13-9c6a-b6ae145ce498.png)

Reply:
@kevinpapst I've opened an issue about that: https://github.com/kevinpapst/TablerBundle/issues/7
If you can paste your reply there. I don't want to poluate  this review with IDE configs.